$MirOS: ports/devel/cvs/patches/patch-src_subr_c,v 1.1 2006/10/02 05:59:18 tg Exp $
--- src/subr.c.orig	Sat Mar 10 22:47:56 2007
+++ src/subr.c	Sat Mar 10 22:47:46 2007
@@ -1285,7 +1285,7 @@ format_cmdline (const char *format, ...)
 	    		dellist(&pflist);
 	    		free(b);
 			error (1, 0,
-"internal error:  unknown integer arg size (%d)",
+"internal error:  unknown integer arg size (%zd)",
                                length);
 			break;
 		}
@@ -1328,7 +1328,7 @@ format_cmdline (const char *format, ...)
 	    		dellist(&pflist);
 	    		free(b);
 			error (1, 0,
-"internal error:  unknown floating point arg size (%d)",
+"internal error:  unknown floating point arg size (%zd)",
                                length);
 			break;
 		}
